www/index.html

```diff
@@ -528,6 +528,25 @@ <h3>Expert settings</h3>
                         </div>
                     </div>
                     <br />
+                    <div class="container">
+                        <h3>Extra Features</h3>
+                        <div class="card" id="extraFeatureDiv">
+                            <div class="card-header">Use Home Key to focus search bar</div>
+                            <div class="card-body">
+                                <div class="checkbox">
+                                    <label>
+                                        <input type="checkbox" name="useHomeKeyToFocusSearchBar"
+                                               id="useHomeKeyToFocusSearchBarCheck">
+                                        <strong>Activate</strong>
+                                    </label>
+                                </div>
+                                <p>
+                                    (Caution!! May produce side effect for few ZIM files in ServiceWorker mode.)
```

Comment:
The side effects should occur rarely, and only on specific ZIM files.
We can probably reword this to something less scary like `(may rarely have side effects on ZIM files that also handle the home key)`

Reply:
I don't think we need a whole new section for this setting, and "Extra Features" is not very useful as a description of the section. This is actually a User Interface Setting, and we currently include UI settings under Display settings.

We could either rename "Display Settings" to "User Interface Settings", or else accept that "Display Settings" includes UI features as we currently do ("Animate transition between app pages" is a UI setting, not strictly a "display" setting).

Personally, I think "Display Settings" is fine for this purpose, and that we should put this option there. It shouldn't have a label "Activate" - the label should describe the setting, i.e. the label should be "Use Home Key to focus search bar".

I agree with @mossroy's wording for the warning.